torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -571,6 +574,56 @@ def test_backend_group(self):
         def test_backend_full_group(self):
             self._test_group_override_backend(self._init_full_group_test)
 
+        # NCCL Batch SEND RECV
+        @skip_if_no_gpu
+        @unittest.skipIf(BACKEND != "nccl", "NCCL Send Recv Only")
```

Comment:
Can we add a test with multiple tags?

Reply:
NCCL doesn't use tags. Do we still want to add that test?